Lacrit/assignment1/q1.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+#include <unordered_map>
+#include <boost/algorithm/string.hpp>
+
+
+// input : string a , string b || output : bool 
+// determine if one is an anagram of another 
+	
+std::unordered_map<char, int> count_occurences(std::string& a, const bool c)
+{
+	//case insensitive
+	if (!c)
+		boost::algorithm::to_lower(a);	
+		
+	//to store letter frequency in the first string
+	std::unordered_map<char,int> count;
+
+	//count char occurences
+	for (size_t i = 0; i < a.length(); i++)
```

Comment:
You could do: `for (char c : a) { ... }` (if you rename the bool parameter, of course)